spec/api-desktop-capturer-spec.js

```diff
@@ -51,4 +51,25 @@ describe('desktopCapturer', () => {
     desktopCapturer.getSources({types: ['window']}, callback)
     desktopCapturer.getSources({types: ['screen']}, callback)
   })
+
+  it('returns an empty screen_api_id for window sources', (done) => {
+    desktopCapturer.getSources({types: ['window']}, (error, sources) => {
+      assert.equal(error, null)
+      assert.notEqual(sources.length, 0)
+      sources.forEach((source) => { assert.equal(source.screen_api_id.length, 0) })
+      done()
+    })
+  })
+
+  it('returns a populated screen_api_id for screen sources on Windows and Mac', (done) => {
+    if (process.platform !== 'win32' && process.platform !== 'darwin') {
+      done()
+    }
+    desktopCapturer.getSources({types: ['screen']}, (error, sources) => {
+      assert.equal(error, null)
+      assert.notEqual(sources.length, 0)
+      sources.forEach((source) => { assert.notEqual(source.screen_api_id.length, 0) })
+      done()
+    })
```

Comment:
Would be awesome to also check that the screen IDs match between the desktop capturer and screen APIs, i.e. enumerate the screens through `screen.getAllDisplays()` and assert that the set of IDs in that list matches the set of IDs from `getSources`.

Reply:
I considered that, but wasn't sure how tightly coupled these unit tests should be. It's an easy change and seems high value, so done!